web/html/src/manager/content-management/list-filters/filter.utils.js

```diff
@@ -126,6 +129,12 @@ export function mapResponseToFilterForm(filtersResponse: Array<FilterServerType>
         filterForm.release = release;
         filterForm.architecture = architecture;
       }
+    } else if(filterResponse.criteriaKey === clmFilterOptions.STREAM.key) {
+      if(!_isEmpty(filterResponse.criteriaValue)) {
+        const [, module, stream] = filterResponse.criteriaValue.match(/(.*)(?::(.*))/);
```

Comment:
I gave it a try and if I just enter a fake name without any `:` in the value, it breaks the filters list page (it does not show anything but the error page) because it cannot return and `Array` of 3 values. We need to make sure the field is populated with valid values.

Reply:
Looks like the missing `:` scenario can be reproduced bycreating a module filter with an empty stream (which should be valid iiuc).

Reply:
Right, sorry, my bad :grimacing: 
This part was not unit tested. Fixed and added tests now.